### PR TITLE
Refine storage package

### PR DESCRIPTION
**Pull request by TY1Fan**
Changes:
* Added EP comments to test cases
* Correct some of the javadocs header comments

Did not spot any commented out or unused code.

### CODE REVIEW

File: src/main/java/seedu/address/storage/JsonAddressBookStorage.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public Optional<ReadOnlyAddressBook> readAddressBook() throws DataLoadingExcepti
     }
 
     /**
-     * Similar to {@link #readAddressBook()}.
+     * Functions similarly to {@link #readAddressBook()}.
```
Comment (Feng1231) [inline src/main/java/seedu/address/storage/JsonAddressBookStorage.java:40]: Thanks for changing these

**Review by Feng1231 — APPROVED** [review]
I think it looks good

**Comment by Feng1231** [comment]
Hi next reviewer, please help to merge if no errors

**Review by DennieDan — APPROVED** [review]
So details! LGTM

**Pull request merged**